src/postgres/commands/deletePostgresServer.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { DialogResponses, IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { PostgresServerTreeItem } from '../tree/PostgresServerTreeItem';
+
+export async function deletePostgresServer(context: IActionContext, node: PostgresServerTreeItem): Promise<void> {
```

Comment:
It should be `node?: PostgresServerTreeItem` since `node` will be undefined when you run this from the command palette. I believe you have a null-ref bug at the moment and fixing the types should cause a build error in that case